src/lib/WazoSessionDescriptionHandler.ts

```diff
@@ -449,7 +449,11 @@ class WazoSessionDescriptionHandler extends SessionDescriptionHandler {
             SessionDescriptionHandler.dispatchRemoveTrackEvent(localStream, oldTrack);
           }
 
-          localStream.addTrack(newTrack);
+          try {
+            localStream.addTrack(newTrack);
+          } catch (e) {
```

Comment:
```suggestion
          } catch (e: any) {
```

It appears we need to specify that type for tests to pass